az-hadoop-jobtype-plugin/src/main/java/azkaban/jobtype/AbstractHadoopJavaProcessJob.java

```diff
@@ -62,10 +62,6 @@ public AbstractHadoopJavaProcessJob(String jobid, Props sysProps, Props jobProps
   public void run() throws Exception {
     try {
       super.run();
-    } catch (Throwable t) {
-      t.printStackTrace();
```

Comment:
sorry, late to the review here, but why are we writing the stacktrace to stderr when we are also logging it via log4j? This call isn't threadsafe.